relay_backend/views/events_stream_with_extras.view.lkml

```diff
@@ -0,0 +1,28 @@
+include: "//looker-hub/relay_backend/views/events_stream_table.view.lkml"
+
+  view: events_stream_with_extras {
+    extends: [events_stream_table]
+
+    dimension: event_id {
+      sql: CONCAT(${TABLE}.document_id, '-',"somestring";;
```

Comment:
This isn't valid SQL (missing closing parenthesis for `CONCAT(...`), and concatenating the document ID with the literal string "something" isn't making a unique event ID.  I'd recommend removing this `event_id` dimension unless it can be properly implemented.